src/azure-cli-core/HISTORY.rst

```diff
@@ -4,9 +4,8 @@ Release History
 ===============
 2.0.59
 ++++++
-* Minor fixes
-
 * Fix issue where in some instances using `--subscription NAME` would throw an exception.
+* Using a random OAuth state for each login request.
```

Comment:
I'm not sure this is really customer-facing enough to justify being in release notes. However, I figured better safe than sorry. Let me know if you'd like me to pull this out, @yugangw-msft.

Reply:
I suggest we pull it out.  To be frank, this code change has zero user impacts. Users won't care.